spec/forms/waste_exemptions_engine/operator_name_form.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
```

Comment:
This filename is missing `_spec` at the end, so RSpec isn't running it and it gets missed in the code coverage.

Reply:
This is actually the case for `operator_postcode_form.rb` and `site_postcode_form.rb` - not from this PR but I just noticed they're missing the `_spec` as well. I think that should bump the coverage up!

Reply:
Thanks for noticing. I'll fix this in a moment.

Reply:
This is fixed in c0b9b1c.

Reply:
I noticed because I've done exactly the same thing!